packages/slate-react/src/components/editor.js

```diff
@@ -2,7 +2,6 @@ import Debug from 'debug'
 import React from 'react'
 import SlateTypes from 'slate-prop-types'
 import Types from 'prop-types'
-import invariant from 'tiny-invariant'
```

Comment:
Is there a good way to test that we caught all the uses of `invariant`?

Reply:
There were just a handful of `invariant` calls in slate according to my search so I just went through them one by one and replaced those that were called with `false` as the first parameter.